scripts/process_output.py

```diff
@@ -307,29 +307,38 @@ def process_metadata(ds):
         The path of a new directory where all data should be stored for this run.
     """
     s = json.dumps(ds.attrs, cls=NumpyEncoder)
+    metadata = json.loads(s) 
     # Hash based on the metadata of the run
-    run_id = hashlib.sha256(s.encode("utf-8")).hexdigest()[:8]
+    hash = hashlib.sha256(s.encode("utf-8")).hexdigest()[:8]    
```

Comment:
hash is a builtin function in Python, so we should probably call this something else to not override it and cause confusion. `hash_digest`?